src/Core/Auth/Models/Business/Tokenables/OrgUserInviteTokenable.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Text.Json.Serialization;
+using Bit.Core.Entities;
+using Bit.Core.Tokens;
+
+namespace Bit.Core.Auth.Models.Business.Tokenables;
+
+public class OrgUserInviteTokenable : ExpiringTokenable
+{
+    // TODO: Ideally this would be internal and only visible to the test project.
+    // but configuring that is out of scope for these changes.
+    public static TimeSpan GetTokenLifetime() => TimeSpan.FromDays(5);
+
+    public const string ClearTextPrefix = "BwOrgUserInviteToken_";
+
+    // Backwards compatibility Note:
+    // Previously, tokens were manually created in the OrganizationService using a data protector
+    // initialized with purpose: "OrganizationServiceDataProtector"
+    // So, we must continue to use the existing purpose to be able to decrypt tokens
+    // in emailed invites that have not yet been accepted.
+    public const string DataProtectorPurpose = "OrganizationServiceDataProtector";
```

Comment:
question: Should we have a tech debt item to follow up and update this purpose?

Reply:
Hmm... actually, after talking with Justin again, we could build in backwards compatibility for the purpose and fallback to the old purpose if the new purpose failed. 

I've created https://bitwarden.atlassian.net/browse/PM-4319 to track this future work.